File: Python/11.loop.sqrt.for.py
n = int(input("Ingresa el n numero de numeros al cuadrado a sumar\n"))
numsqrt = 0

for i in range(1, n + 1):
  print(i, end= "")
  print("^2",end=" ")
  print("+ ",end="")
  numsqrt += i*i
    
print(" ")
print("El resultado es: ",end=" ")
print(numsqrt)

File: C++/11.loop.sqrt.for.cpp
#include <iostream>
using namespace std;
int main() {
	int usri;
	int numsqrt=0;
	cout<<"Ingresa el n numero de numeros al cuadrado a sumar\n";
	cin>>usri;
	for(int i=1;i<=usri;i++){
		numsqrt+=i*i;
		if(i!=usri){cout<<i<<"^2"<<" + ";
		 
		}else cout<<"\nResultado: "<<numsqrt;
}

}
